ertviz/models/plot_model.py

```diff
@@ -124,6 +124,45 @@ def display_name(self):
             return self._name
 
 
+class BarChartPlotModel:
+    def __init__(self, data_df_dict, colors):
+        self._data_df_dict = data_df_dict
+        self._colors = colors
+        self.selection = []
+
+    @property
+    def plot_ids(self):
+        return {plot_idx: ens_name for plot_idx, ens_name in enumerate(self.data)}
+
+    @property
+    def data(self):
+        return self._data_df_dict
+
+    @property
+    def repr(self):
+        fig = go.Figure()
+        response_name = ""
+        for ens_name in self.data:
+            response_name = self.data[ens_name].index.name
```

Comment:
assigning the `self.data[ens_name]` to a variable to avoid all the lookups further down?

Reply:
`self.data` is a dictionary containing only a few entries (it's called only two times), so it's not a big lookup overhead.